src/types.ts

```diff
@@ -288,6 +320,67 @@ type TransactionEvidenceSummary = {
   devicesByAccountTotal30d?: number
 }
 
+type WebTransactionEvidenceSummary = WebEvidenceSummary
+
+type WebEvidenceSummary = {
+  vpn?: VpnEvidence
+  vpnapiIo?: VpnapiIoEvidence
+  bot?: BotEvidence
+  multipleAccounts?: MultipleAccountsEvidence
+  multipleInstallations?: MultipleInstallationsEvidence
+  suspiciousFingerprintVelocity?: SuspiciousFingerprintVelocityEvidence
+  deviceModel?: string
+}
+
+type VpnEvidence = {
+  result?: DetectionResult
+  rttDetectionScore?: number
+  tcpSignatureDetectionScore?: number
+  ipDatabaseDetectionScore?: number
+}
+
+type VpnapiIoEvidence = {
+  result: DetectionResult
+  vpn: DetectionResult
+  proxy: DetectionResult
+  tor: DetectionResult
+  relay: DetectionResult
+}
+
+type BotEvidence = {
+  result?: DetectionResult
+}
+
+type MultipleAccountsEvidence = {
+  accountsByDeviceTotal15d?: number
+  accountsByDeviceTotal30d?: number
+  accountsByDeviceTotal60d?: number
+  accountsByInstallationTotal15d?: number
+  accountsByInstallationTotal30d?: number
+  accountsByInstallationTotal60d?: number
+  accountsByIpTotal15d?: number
+}
+
+type MultipleInstallationsEvidence = {
+  result?: DetectionResult
+  installationsByIpTotal15d?: number
+}
+
+type SuspiciousFingerprintVelocityEvidence = {
```

Comment:
This evidence group also has a `result` field.

Reply:
Thanks Gio!